rules/sinks/third_parties/sdk/firebaseapp/javascript.yaml

```diff
@@ -11,3 +11,59 @@ sinks:
     patterns:
       - "ngx-auth-firebaseui|@asithade\\/ngx-auth-firebaseui"
     tags:
+
+  - id: ThirdParties.SDK.Firebaseapp.React-firebase-hooks
```

Comment:
@khemrajrathore are we ok with this naming convention? I think capitalisation works better than using hyphens. What are your thoughts?

Reply:
Yes, I agree, having a standard naming convention is essential. Let's follow the capitalized convention

Reply:
ok